File: seleniumTestXpath.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using System.Threading;
class EntryPoint
{
    static void Main()
    {
        //*[@id="post-108"]/div/figure/img
        string url = "https://testing.todorvachev.com/css-path/";
        string xPath = "//*[@id='cat']";

        IWebDriver driver = new ChromeDriver();
        
        driver.Navigate().GoToUrl(url);

        IWebElement element = driver.FindElement(By.XPath(xPath));

        if (element.Displayed)
        {
            Console.WriteLine("Yes, it's displayed!");
        } else
        {
            Console.WriteLine("No, it's not displayed...");
        }

        Thread.Sleep(7000);

        driver.Quit();

    }
}

File: Java/SeleniumTestXpath.java
package com.selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Main {

    public static void main(String[] args) throws InterruptedException {
	// write your code here
        String url = "https://testing.todorvachev.com/css-path/";
        String xPath = "//*[@id='cat']";

        WebDriver driver = new ChromeDriver();

        driver.get(url);

        WebElement element = driver.findElement((By.xpath(xPath)));

        if (element.isDisplayed())
        {
            System.out.println(("Yes, it's displayed!"));
        } else
        {
            System.out.println("No, it's not displayed...");
        }

        Thread.sleep(7000);

        driver.quit();
    }
}